readthedocs/templates/projects/domain_list.html

```diff
@@ -33,16 +44,18 @@ <h3> {% trans "Existing Domains" %} </h3>
   </p>
   {% endif %}
 
-  <h3> {% trans "Add new Domain" %} </h3>
-
-  {% if not enabled %}
-    {% include 'projects/includes/feature_disabled.html' with project=project %}
-  {% else %}
-    <form method="post" action="{% url 'projects_domains_create' project.slug %}">{% csrf_token %}
-      {{ form.as_p }}
-      <p>
-        <input style="display: inline;" type="submit" value="{% trans "Add" %}">
-      </p>
-    </form>
+  {% if not superproject %}
+    <h3>{% trans "Add new Domain" %}</h3>
```

Comment:
Might as well update while changing this:

```suggestion
    <h3>{% trans "Add new domain" %}</h3>
```

Based on style discussions on blog content and site content, headings deeper than `h1` can be sentence case.